lib/include/openamp/virtio_mmio.h

```diff
@@ -167,6 +168,70 @@ struct virtio_mmio_device {
 	void *user_data;
 };
 
+/**
```

Comment:
I guess the code below should be under `#ifdef WITH_VIRTIO_MMIO_DEV`. it should not compile if not defined